packages/react-hooks/README.md

```diff
@@ -84,3 +84,19 @@ function MockComponent() {
   );
 }
 ```
+
+### `usePrevious()`
+
+This hook will store the previous value of a given variable.
+
+```tsx
+function Score({value}) {
+  const previousValue = usePrevious(value);
+  return (
+    <>
+      <p>Current score: {value}</p>
+      {value > previousValue && <p>We have a new record!</p>}
```

Comment:
Nit. This coding style feels a bit off from how we generally handle conditionally rendering content.  I would suggest pulling it out 

```tsx
const newRecord = value > previousValue ? <p>We have a new record!</p> : null;

return ( 
...
  {newRecord}
...
) 
```

Reply:
Done!